agent/wsclient/mock/utils/utils.go

```diff
@@ -22,9 +22,9 @@ import (
 	"github.com/gorilla/websocket"
 )
 
-// StartMockServer starts a mock websocket server.
+// GetMockServer retuns a mock websocket server that can be started up as TLS or not.
 // TODO replace with gomock
-func StartMockServer(t *testing.T, closeWS <-chan []byte) (*httptest.Server, chan<- string, <-chan string, <-chan error, error) {
+func GetMockServer(t *testing.T, closeWS <-chan []byte) (*httptest.Server, chan<- string, <-chan string, <-chan error, error) {
```

Comment:
"Should we gate this code path with an 'allowUnsecureACS' option?"

So my argument against is that we do allow using an http endpoint to talk to ECS via the SDK. If there were a flag, it should apply to both.